doc/README.md

```diff
@@ -38,7 +38,7 @@ Building
 The following are developer notes on how to build Bitcoin on your native platform. They are not complete guides, but include notes on the necessary libraries, compile flags, etc.
 
 - [Dependencies](dependencies.md)
-- [OS X Build Notes](build-osx.md)
+- [macOS Build Notes](build-macOS.md)
```

Comment:
No need to rename the file every time apple changes the name of the operating system. This will break all external links.

Reply:
Good point! 👍 I have reverted the change in [2d7a92c](https://github.com/bitcoin/bitcoin/pull/13366/commits/2d7a92c8f7ae0e40d48cf2464073c357b33763e6)